### PR TITLE
Add WebAssembly

### LINKED ISSUE #1165: Awesome wasm
[mbasso/awesome-wasm](https://github.com/mbasso/awesome-wasm) - Awesome WebAssembly (wasm) list.
I did not find a mention of wasm, and also WebAssembly in the repository, I suggest adding this repository to the list, it is currently the only list.
I also suggest discussing what category the technology falls into.

### PR DESCRIPTION
**Pull request by mbasso**
**https://github.com/mbasso/awesome-wasm**

**This resolves #1165, I think that this list might be a good starting point to learn WebAssembly and find some interesting projects, articles and so on about it**

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- Not a duplicate.
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
		- Should be placed on the right side of the readme heading.
		- Should link back to this list.
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
		- Should only have one level of sub-lists, preferably none.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
		- The description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Please read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) more closely. Multiple points not followed from it. Try going through the list point for point and check that you follow it. :)

**Comment by sindresorhus** [comment]
Ping. Please also see #1363.

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/mbasso/awesome-wasm
✖ Linting

  README.md:307:3
  ✖    1:1   Missing content headers                                       remark-lint:awesome/toc
  ✖    1:1   License section must be at heading depth 2                    remark-lint:awesome/license
  ✖    1:17  Invalid badge source                                          remark-lint:awesome/badge
  ✖   20:6   Text "node.js" should be written as "Node.js"                 remark-lint:awesome/spell-check
  ✖   54:4   Text "Github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  132:6   Text "node.js" should be written as "Node.js"                 remark-lint:awesome/spell-check
  ✖  226:77  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  290:4   Text "StackOverflow" should be written as "Stack Overflow"    remark-lint:awesome/spell-check
  ✖  307:3   Don’t pad link with inner spaces                              remark-lint:no-inline-padding

  9 errors
```

https://github.com/sindresorhus/awesome-lint

File: readme.md [hunk 1]
```diff
@@ -166,6 +166,7 @@
 - [ActionScript 3](https://github.com/robinrodricks/awesome-actionscript3) - Object-oriented language targeting Adobe AIR.
 - [Eta](https://github.com/sfischer13/awesome-eta) - Functional programming language for the JVM.
 - [Idris](https://github.com/joaomilho/awesome-idris) - General purpose pure functional programming language with dependent types influenced by Haskell and ML.
+- [WebAssembly](https://github.com/mbasso/awesome-wasm)
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should:
* Include a short description about the project/theme of the list. **It should not describe the list itself.**

**Comment by sindresorhus** [comment]
// @mbasso @MaxGraey

**Comment by mbasso** [comment]
Hi @sindresorhus,
I have fixed merge conflicts, item description and the linting but I've not done the tasks defined in #1363 yet.

**Comment by mbasso** [comment]
I'm currently following #1547 and #1545

**Comment by sindresorhus** [comment]
`Awesome Wasm` => `Awesome WebAssembly` (The heading)

---

  ✖  215:3  Link to https://gist.github.com/icefox/e58d23e860a0b525e0044cac120f667b is dead  remark-lint:no-dead-urls

---

> Includes a succinct description of the project/theme at the top of the readme. (Example)

---

The description part of entries should not be part of the link. For example, only `cpp-wasm-loader` in `cpp-wasm-loader - C/C++ to WASM Webpack loader optimized for small bundle sizes` should be a link.

---

> Entries have a description, unless the title is descriptive enough by itself. It rarely is though.

---

Includes a project logo/illustration whenever possible.
Either centered, fullwidth, or placed at the top-right of the readme. (Example)
The image should link to the project website or any relevant website.
The image should be high-DPI. Set it to maximum half the width of the original image.

File: readme.md [hunk 2]
```diff
@@ -166,6 +166,7 @@
 - [Idris](https://github.com/joaomilho/awesome-idris#readme) - General purpose pure functional programming language with dependent types influenced by Haskell and ML.
 - [Ada/SPARK](https://github.com/ohenley/awesome-ada#readme) - Modern programming language designed for large, long-lived apps where reliability and efficiency are essential.
 - [Q#](https://github.com/ebraminio/awesome-qsharp#readme) - Domain-specific programming language used for expressing quantum algorithms.
+- [WebAssembly](https://github.com/mbasso/awesome-wasm) - A binary instruction format for a stack-based virtual machine.
```
Comment (sindresorhus) [inline readme.md]: I think the Google description is more useful:

> A standard that defines a portable binary format and a corresponding assembly-like text format for executables.

**Comment by sindresorhus** [comment]
Ping

**Pull request closed without merging**